apps/frontend/src/api/util.ts

```diff
@@ -43,3 +47,26 @@ export const useFetchOptions = ({
     },
   };
 };
+
+export interface BasicOptions {
+  searchQuery?: string;
+  filters?: string[];
+}
+
+export const useApiGet = <T>(
+  endpoint: string,
+  parameters: Record<string, string | string[] | undefined> = {},
+  options?: Parameters<typeof useFetchOptions>[0],
+) => {
+  const url = new URL(`${API_BASE_URL}${endpoint}`);
```

Comment:
```suggestion
  const url = new URL(endpoint, API_BASE_URL);
```

Reply:
I did this initially but there is /development on the API_BASE_URL

Reply:
Okay, but then the problem is really that `endpoint` is absolute.
Can we change the paths so that my solution effectively does `new URL('users', 'https://host/development/')`? (which works)